### PR TITLE
AnimeSaturn fixed

### LINKED ISSUE #3462: [Presence | Bug] AnimeSaturn
**Describe the Bug**

The links of Anime Saturn are not detected
Example:
https://www.animesaturn.it/watch
When I open this kind of URL the extension doesn't detect that it is an AnimeSaturn URL

It only happens if that page is opened without going to the main site
Example:
https://www.animesaturn.it > https://www.animesaturn.it/anime/ > https://www.animesaturn.it/watch (works)
https://www.animesaturn.it/watch (doesn't work)
**Steps to Reproduce**

1.use a direct link to a page
https://www.animesaturn.it/watch

without going to the main page
https://www.animesaturn.it > https://www.animesaturn.it/anime/ > https://www.animesaturn.it/watch
2.
3.

**Expected Behavior**

venga rilevata la pagina
https://www.animesaturn.it/watch
come pagina di anime saturn e venga aggiornato lo stato Discord

**Screenshots**

**Environment**

- OS: Windows 10
- Browser: Opera Gx
- Version: LVL 2 (core: 73.0.3856.424)
  

**Additional context**

it's not too important a problem, but for me who often use direct links to see anime it is a bit broken

### PR DESCRIPTION
**Pull request by DarkVillager**
Resolves #3462
- New dom for watching anime
- Taking over cause not maintained (issues open since Apr 8 2021)
image

### CODE REVIEW

File: websites/A/AnimeSaturn/dist/metadata.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
   "author": {
+    "name": "Dark_Ville",
+    "id": "638080361179512853"
+  },
+  "contributor": {
     "name": "Tellen",
     "id": "535214819520020510"
   },
```
Comment (Bas950) [inline websites/A/AnimeSaturn/dist/metadata.json:9]: contributors should be in an array
and why even change of author?

File: websites/A/AnimeSaturn/presence.ts [hunk 2]
```diff
@@ -11,8 +11,8 @@ function getTimestamps(
   videoTime: number,
   videoDuration: number
 ): Array<number> {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  const startTime = Date.now(),
+   endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
```
Comment (Bas950) [inline websites/A/AnimeSaturn/presence.ts:15]: use presence.getTimestamps

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/AnimeSaturn/dist/metadata.json (hunk 1 above)
Comment (DarkVillager) [reply websites/A/AnimeSaturn/dist/metadata.json]: Taking over cause not maintained (issues open since Apr 8 2021)

**Comment by DarkVillager** [comment]
I can’t work on this till Monday just to let the mods know.

File: websites/A/AnimeSaturn/dist/metadata.json (hunk 1 above)
Comment (Bas950) [reply websites/A/AnimeSaturn/dist/metadata.json]: Did you contact the maintainer tho?

File: websites/A/AnimeSaturn/dist/metadata.json [hunk 3]
```diff
@@ -1,8 +1,14 @@
 {
   "author": {
-    "name": "Tellen",
-    "id": "535214819520020510"
+    "name": "Dark_Ville",
+    "id": "638080361179512853"
   },
+  "contributor": [
```
Comment (Slowlife01) [inline websites/A/AnimeSaturn/dist/metadata.json:6]:
```suggestion
  "contributors": [
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

**Comment by Slowlife01** [comment]
Maybe change the client id as well

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
You reference `presence.setActivity(data)` sixteen times within this presence, refactor that down?

**Comment by Slowlife01** [comment]
Inactive

**Pull request closed without merging**